src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -219,8 +223,9 @@ static bool CollectCrashDump(Process process, string path)
             else if (OperatingSystem.IsLinux() || OperatingSystem.IsMacOS())
             {
                 createdump.StartInfo.FileName = "sudo";
```

Comment:
It would be great if this could run without sudo

Reply:
What if the process is started by create dump? Does that change anything?

Reply:
It looks like ptrace_scope is 1 by default, meaning that parent processes should be able to attach to child processes.

Reply:
This portion is not touched by the PR, so I would rather have a separate PR to factor this out.